.github/workflows/main.yml

```diff
@@ -883,5 +883,5 @@ jobs:
         run: |
           go env -w GOFLAGS=-mod=mod
           go install github.com/google/addlicense@2fe3ee94479d08be985a84861de4e6b06a1c7208
-          addlicense -ignore "**/script-empty.sh" -ignore "testdata/**" -ignore "**/testdata/**" -l apache -c 'Security Scorecard Authors' -v *
-          git diff --exit-code
+          addlicense -s -ignore "**/script-empty.sh" -ignore "pkg/testdata/**" -ignore "checks/testdata/**" -l apache -c 'OpenSSF Authors' -v *
```

Comment:
what's the `s` option for?
Why shall we change the `-ignore` list? if we do that, every time we add a testdata folder it will complain; whereas now it catches existing ones and the one we create in the future. Right?

Updating `Security Scorecard` to `OpenSSF` is fine, but you're going to have to update all the files.

Is this change a recommendation from OpenSSF?

Reply:
`-s`  to addlicense to generate SPDX header. 
I have made these changes before in [PR-1718](https://github.com/ossf/scorecard/pull/1718) against the issue [Handling dates in copyright headers #1534](https://github.com/ossf/scorecard/issues/1534). It is not part of current changes. My apologies, These changes came by mistake. I will revert the changes.
Thank You

Reply:
@singhsaurabh any reason to change `**/testdata/**` to `checks/testdata/**`? I think the former catches all testdata folders, rather than just the ones under checks.